asv_bench/benchmarks/benchmarks.py

```diff
@@ -363,6 +363,19 @@ def time_sort_values(self, shape, columns_number, ascending_list):
         execute(self.df.sort_values(self.columns, ascending=self.ascending))
 
 
+class TimeAstype:
+    param_names = ["shape"]
+    params = [
+        UNARY_OP_DATA_SIZE[ASV_DATASET_SIZE],
+    ]
+
+    def setup(self, shape):
+        self.df = generate_dataframe(ASV_USE_IMPL, "int", *shape, RAND_LOW, RAND_HIGH)
+
+    def time_astype(self, shape):
+        execute(self.df.astype("float64"))
+
```

Comment:
I think that we should trigger execution with `.dtype` here, because for some types `new_dtypes` will be set as `None`, and will be computed only when `.dtype` property will be accessed next time
https://github.com/modin-project/modin/blob/3a3168382fb18f2a41de8858bac0f566415946a9/modin/engines/base/frame/data.py#L729-L732

Reply:
And I think we also should bench `.astype("category")` since this behaves differently from another types

Reply:
Added cast to `category` type and usage `dtypes` as a trigger of computation.